packages/cbor/lib/decoder.js

```diff
@@ -79,6 +79,8 @@ class UnexpectedDataError extends Error {
  * @property {boolean} [extendedResults=false] If true, emit extended
  *   results, which will be an object with shape {@link ExtendedResults}.
  *   The value will already have been null-checked.
+ * @property {boolean} [allowMapsWithDuplicateKeys=true] If false, error is
```

Comment:
I think this would be better as `preventDuplicateKeys` so that the default can be false.